packages/url_launcher/url_launcher/README.md

```diff
@@ -49,6 +49,42 @@ void _launchURL() async =>
     await canLaunch(_url) ? await launch(_url) : throw 'Could not launch $_url';
 ```
 
+### Android
+
+Starting from API 30 Android requires package visibility configuration in your `AndroidManifest.xml` otherwise `canLaunch` would return `false`. Please see the `<queries>` element example below and tailor to your needs. For more information see the [Managing package visibility documentation](https://developer.android.com/training/basics/intents/package-visibility).
+
+``` xml
+<manifest ...>
+
+  ...
+
+  <queries>
+    <!-- If your app opens https URLs -->
+    <intent>
+      <action android:name="android.intent.action.VIEW" />
+      <data android:scheme="https" />
+    </intent>
+    <!-- If your app makes calls -->
+    <intent>
+      <action android:name="android.intent.action.DIAL" />
+      <data android:scheme="tel" />
+    </intent>
+    <!-- If your app emails -->
+    <intent>
+      <action android:name="android.intent.action.SEND" />
+      <data android:mimeType="*/*" />
+    </intent>
+  </queries>
+
+  ...
+
+  <application>
```

Comment:
Why include this here just to elide the contents? These four lines don't seem to be adding any clarity.

Reply:
Often times if I just show an element without the surroundings people won't know right away if it should be placed inside the <application> tag, or as a root element child. Before or after the <application>, etc. The surrounding would make it more straight forward for some, but if someone copy-pastes the whole snippet the ellipsis can cause trouble. I'm rephrasing the text right now so it'll be more descriptive.

Reply:
I took out the ellipsis and the `<application>` BTW.